### ISSUE TITLE
Provide link next to shares

### ISSUE THREAD

**Issue by tbsbdr** [owncloud/web#9266]
![screenshot_001240](https://github.com/owncloud/web/assets/26610733/ff98aad6-9a63-4bec-afd6-ae7c1783d8c5)

**Comment by kulmann** [comment]
https://github.com/owncloud/ocis/issues/6120 needs to be done first

**Comment by tbsbdr** [comment]
duplicate of https://github.com/owncloud/web/issues/9376
